JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -33,12 +33,13 @@ StatusSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that obse
 ControlSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,"A device that is used to measure, monitor, and control processes within various industrial and commercial applications.",https://www.theworldavatar.com/kg/ontodevice/,Control sensor
 FlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is used to measure the flow rate of fluids.,https://www.theworldavatar.com/kg/ontodevice/,Flow sensor
 FeedbackSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is used to provide the control system with measurements of physical quantities necessary to close control loops.,https://www.theworldavatar.com/kg/ontodevice/,Feedback sensor
+ProximitySensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that takes a continuous distance reading and returns a corresponding analog voltage.,https://www.theworldavatar.com/kg/ontodevice/,Proximity sensor
 Fridge,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that is usually used for the preservation of items like food via temperature regulation.,https://www.theworldavatar.com/kg/ontodevice/,Fridge
 Setpoint,Class,,,,,,"The desired or target value for an essential variable, or process value (PV) of a control system.",https://www.theworldavatar.com/kg/ontodevice/,Setpoint
 InOutState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of something on whether it is inside or outside of a space or object.,https://www.theworldavatar.com/kg/ontodevice/,In out state
+OccupiedUnoccupiedState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of something on whether it is occupied or not.,https://www.theworldavatar.com/kg/ontodevice/,Occupied unoccupied state
 PowerFactor,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,"The ratio of working power, measured in kilowatts (kW), to apparent power, measured in kilovolt amperes (kVA).",https://www.theworldavatar.com/kg/ontodevice/,Power factor
-ScreenBrightness,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,The brightness of the screen of a device measured as a ratio.,https://www.theworldavatar.com/kg/ontodevice/,Screen brightness
-Property,Class,,,,,,An attribute of a device or space that is being measured by another device.,https://www.theworldavatar.com/kg/ontodevice/,Property
+Brightness-Ratio,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,Brightness measured as a ratio.,https://www.theworldavatar.com/kg/ontodevice/,Brightness-Ratio
```

Comment:
Would RelativeBrightness be a better name (analogous to RelativeHumidity)? I assume you'd expect values between zero and one?